portal/settings.py

```diff
@@ -102,7 +105,15 @@
     },
 ]
 
+AUTHENTICATION_BACKENDS = (
+   # 'social_core.backends.open_id.OpenIdAuth',  # for Google authentication
+   # 'social_core.backends.google.GoogleOpenId',  # for Google authentication
+   # 'social_core.backends.google.GoogleOAuth2',  # for Google authentication
+    'social_core.backends.github.GithubOAuth2',  # for Github authentication
+    #'social_core.backends.facebook.FacebookOAuth2',  # for Facebook authentication
```

Comment:
Please remove everything except Github, as there is never a case anything else would be required.

Reply:
done